pkg/security/scc/byrestrictions_test.go

```diff
@@ -168,3 +170,94 @@ func TestVolumePointValue(t *testing.T) {
 		}
 	}
 }
+
+func TestCapabilitiesPointValue(t *testing.T) {
+	newSCC := func(def []kapi.Capability, allow []kapi.Capability, drop []kapi.Capability) *securityapi.SecurityContextConstraints {
+		return &securityapi.SecurityContextConstraints{
+			DefaultAddCapabilities:   def,
+			AllowedCapabilities:      allow,
+			RequiredDropCapabilities: drop,
+		}
+	}
+
+	tests := map[string]struct {
+		defaultAdd     []kapi.Capability
+		allowed        []kapi.Capability
+		requiredDrop   []kapi.Capability
+		expectedPoints points
+	}{
+		"nothing specified": {
+			defaultAdd:     nil,
+			allowed:        nil,
+			requiredDrop:   nil,
+			expectedPoints: capDefaultPoints,
+		},
+		"default": {
+			defaultAdd:     []kapi.Capability{"KILL", "MKNOD"},
+			allowed:        nil,
+			requiredDrop:   nil,
+			expectedPoints: capDefaultPoints + 2*capAddOnePoints,
+		},
+		"allow": {
+			defaultAdd:     nil,
+			allowed:        []kapi.Capability{"KILL", "MKNOD"},
+			requiredDrop:   nil,
+			expectedPoints: capDefaultPoints + 2*capAllowOnePoints,
+		},
+		"allow star": {
+			defaultAdd:     nil,
+			allowed:        []kapi.Capability{"*"},
+			requiredDrop:   nil,
+			expectedPoints: capDefaultPoints + capAllowAllPoints,
+		},
+		"allow all": {
+			defaultAdd:     nil,
+			allowed:        []kapi.Capability{"ALL"},
+			requiredDrop:   nil,
+			expectedPoints: capDefaultPoints + capAllowAllPoints,
+		},
+		"allow all case": {
+			defaultAdd:     nil,
+			allowed:        []kapi.Capability{"All"},
+			requiredDrop:   nil,
+			expectedPoints: capDefaultPoints + capAllowAllPoints,
+		},
+		"drop": {
+			defaultAdd:     nil,
+			allowed:        nil,
+			requiredDrop:   []kapi.Capability{"KILL", "MKNOD"},
+			expectedPoints: capDefaultPoints + 2*capDropOnePoints,
+		},
+		"drop all": {
+			defaultAdd:     nil,
+			allowed:        nil,
+			requiredDrop:   []kapi.Capability{"ALL"},
+			expectedPoints: capDefaultPoints + capDropAllPoints,
+		},
+		"drop all case": {
+			defaultAdd:     nil,
+			allowed:        nil,
+			requiredDrop:   []kapi.Capability{"all"},
+			expectedPoints: capDefaultPoints + capDropAllPoints,
+		},
+		"drop star": {
```

Comment:
I've turned it into checking that `'*'` is treated as single capability while dropping, just like any other string -> 3f0fe0e425b9786b658c5e8c025ec2ead1a45569.